src/stories/components/svg/ArrowInternalLink.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+interface Props {
+  width?: number;
+  height?: number;
+  className?: string;
+  fill?: string;
+}
+
+const ArrowInternalLink: React.FC<Props> = ({ className, width = 24, height = 24, fill = '#5B667E', ...props }) => (
```

Comment:
this is a new component so it should be in the new component folder

Reply:
you are right, done